scenarios/multi_object_tracking/01_training_introduction.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<ul><li>[minor] All annotation instructions for video could be in the FAQ (including the 30fps). </li><li>Does VOTT output annotations in this format? <code>[image] [xmin] [y_min] [x_max] [y_max] [label]</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>